docs/source/index.rst

```diff
@@ -6,36 +6,58 @@
 Substra documentation
 =====================
 
-Substra is a framework offering distributed orchestration of machine learning tasks among partners while guaranteeing secure and trustless traceability of all operations.
+Substra is an open source federated learning framework. It provides a flexible python interface to run federated learning training at scale.
```

Comment:
isn't it restrictive to mention only a python interface?

Reply:
Oups forgot the front.
Added